Dockerfile

```diff
@@ -38,3 +37,5 @@ RUN apk --no-cache add ca-certificates
 RUN mkdir -p /opt/resource
 
 COPY --from=builder /root/go/bin/* /opt/resource/
+
+RUN ln -s /opt/resource/cogito /opt/resource/check
```

Comment:
You can probably spare a few more bytes by hard-linking it instead of using a symlink. :smile: